### PR TITLE
[#8589] Student Profile: add a 'Upload/Edit Photo' tooltip

### LINKED ISSUE #8589: Student Profile: add a 'Upload/Edit Photo' tooltip
This tooltip will show size and formats of a photo. It would be easier for a student to look for available photos to submit.

There is a similar tooltip (shown in the screenshot below), but it also can be changed. For example, the new tooltip will look like this 'Upload a profile picture. Max size: 5MB .png .jpg'.
profile picture

### PR DESCRIPTION
**Pull request by YaroslavGnatyuk**
Fixes #8589

_Updated student profile tooltip for photo according to ticket description._

### CODE REVIEW

**Comment by YaroslavGnatyuk** [comment]
Hello! Could you help me? I found in a guide how to run a single test. But I want to run a single test in debugging. How can I do it? How can I run project or single test in debuggin resume? I ask because I have failed unit tests during CI build.

**Comment by sukanta-27** [comment]
@YaroslavGnatyuk As you have made UI changes, you will have to run [GodMode](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md) on the tests that are failing due to your changes.

**Comment by YaroslavGnatyuk** [comment]
Thanks, do I have possibility to run tests in debug mode?

**Comment by sukanta-27** [comment]
@YaroslavGnatyuk Yes.

**Comment by YaroslavGnatyuk** [comment]
@sukanta-27 Could please explain how to run project or unit tests(test) in debug mode?

**Comment by YaroslavGnatyuk** [comment]
@sukanta-27 Could you please review my solution?

**Comment by sukanta-27** [comment]
@YaroslavGnatyuk Can add a screenshot first? (As you are doing a UI change, it will help if you add a screenshot of your fix so that we know it is working)

**Comment by YaroslavGnatyuk** [comment]
@sukanta-27 
screenshot from 2018-08-06 07-23-50

**Comment by YaroslavGnatyuk** [comment]
According to issue as I understand tooltip belong to profile pic. But if you think that makes more sense change tooltip for button I can update my fix. How do you think?

**Comment by YaroslavGnatyuk** [comment]
@sukanta Ok, thank you. I I'm going to update the fix today.

**Comment by YaroslavGnatyuk** [comment]
@sukanta-27 Hi. Updated fix according to your suggestion.
Screenshot attached:

screenshot from 2018-08-12 15-47-54

**Comment by YaroslavGnatyuk** [comment]
@sukanta-27 Hello! I have already updated fix. Could you please review my solution?
Also, screenshot attached: 
screenshot from 2018-08-12 15-47-54

File: src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag [hunk 1]
```diff
@@ -13,18 +13,18 @@
...
+    <div class="col-xs-6 col-sm-5 col-md-3 cursor-pointer">
       <img id="profilePic"
           src="${profile.pictureUrl}"
           class="profile-pic"
           data-toggle="modal"
           data-target="#studentPhotoUploader"
           data-edit="${profile.editingPhoto}">
     </div>
-    <div class="">
+    <div class="col-xs-6 col-sm-5 col-md-3 cursor-pointer"
```
Comment (sukanta-27) [inline src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: I think the `cursor-pointer` here is unnecessary.

Comment (YaroslavGnatyuk) [reply src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: Ok, thaks

Comment (YaroslavGnatyuk) [reply src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: @sukanta-27 Done

Comment (YaroslavGnatyuk) [reply src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag]: @sukanta-27 Done, but the latest build was failed. Something went wrong with - "nvm install node" failed and exited with 3 during.

**Comment by YaroslavGnatyuk** [comment]
@sukanta-27 CI build finished without errors. What I should to do to merge the fix?

**Comment by YaroslavGnatyuk** [comment]
So can I take another issue not for beginners? For example some unit tests?

**Comment by sukanta-27** [comment]
>So can I take another issue not for beginners? For example some unit tests?

Go ahead. :+1:

**Comment by YaroslavGnatyuk** [comment]
Ok, thanks!

**Pull request merged**